community/cloud-foundation/templates/runtime_config/README.md

```diff
@@ -0,0 +1,76 @@
+# Google Cloud Runtime Configurator
+
+This template creates a Runtime Configurator with the associated resources.
+
+## Prerequisites
+
+- Install [gcloud](https://cloud.google.com/sdk)
+- Install gcloud **beta** components:
+
+  ```shell
+  gcloud components update
+  gcloud components install beta
+  ```
+
+- Create a [GCP project, set up billing, enable requisite APIs](../project/README.md)
+- Enable the [Cloud Runtime Configurator API](https://console.developers.google.com/apis/api/runtimeconfig.googleapis.com)
+- Grant the [Cloud RuntimeConfig Admin](https://console.cloud.google.com/iam-admin/roles/details/roles%3Cruntimeconfig.admin)
```

Comment:
The link seems to load a broken page, unless the query parameter "?project_id" and "&organization_id" are specified. Should we replace this with a link to the docs here: https://cloud.google.com/deployment-manager/runtime-configurator/access-control ?

Reply:
Good catch. Fixed in `d23be2f`